src/components/TotalContributions/TotalContributions.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
+import ContribsChart from './ContribsChart';
 
 function TotalContributions(props){
 
   return (
     <div>
-      <h3>total donations in most recent cycle: ${props.contribs.total_donations.toLocaleString()}</h3>
-      <h3>spent in most recent cycle: ${props.contribs.spent.toLocaleString()}</h3>
-      <h3>Cash that the campaign has on hand: ${props.contribs.cash_on_hand.toLocaleString()}</h3>
+      <ContribsChart donations={props.contribs.total_donations} spent={props.contribs.spent}/>
```

Comment:
Where is props.contribs.spent coming from

Reply:
Should be coming from the individual rep component